uservice_ccutter/plugins/projecttypes/lsst_technote_bootstrap.py

```diff
@@ -468,11 +329,11 @@ def _enable_protected_branches(auth, inputdict):
     _debug("Changing protection with URL %s" % prot_url)
     # Sometimes this, weirdly, gets a 404.  We'll wrap it in a retry
     #  loop
-    req = _retry_request("put", prot_url, headers=headers, payload=data,
-                         auth=(user, token))
-    raise_from_req(req)
+    req = retry_request("put", prot_url, headers=headers, payload=data,
+                        auth=(user, token))
+    raise_from_response(req)
 
 
 def _debug(*args):
     if os.environ.get("DEBUG"):
-        print(*args)
+        print("DEBUG:", *args)
```

Comment:
I'm continuing to send shade toward this `_debug` method. Python logging ftw. Looking forward to seeing that JIRA issue implemented.